app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/pullrequests/PullRequestViewModel.kt

```diff
@@ -1,23 +1,28 @@
 package com.bottlerocketstudios.brarchitecture.ui.pullrequests
 
-import androidx.lifecycle.viewModelScope
 import com.bottlerocketstudios.brarchitecture.data.repository.BitbucketRepository
 import com.bottlerocketstudios.brarchitecture.domain.models.Status
 import com.bottlerocketstudios.brarchitecture.ui.BaseViewModel
 import com.bottlerocketstudios.compose.pullrequest.PullRequestItemState
 import com.bottlerocketstudios.compose.util.asMutableState
 import com.bottlerocketstudios.compose.util.formattedUpdateTime
+import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.map
-import kotlinx.coroutines.launch
 import org.koin.core.component.inject
 import java.time.Clock
+import java.util.Locale
+
+enum class PRState { OPEN, MERGED, DECLINED, SUPERSEDED }
```

Comment:
Since we're not using this enum yet, go ahead and pull it.  We'll create some magick soon!

Reply:
@BottleRocket-Colin I would think that this enum could be mapped from the bitbucket pullrequests api state value in the response:

> state
> string
>
> Only return pull requests that are in this state. This parameter can be repeated.
>
> Valid values: OPEN, MERGED, DECLINED, SUPERSEDED
> Source: https://developer.atlassian.com/cloud/bitbucket/rest/api-group-pullrequests/#api-pullrequests-selected-user-get

1. We could map from the `PullRequestDto.state` `String?` to `PrState` in a separate Converter file (similar to other files in `com.bottlerocketstudios.brarchitecture.data.converter`).
2. Then `PullRequestItemState.prState` could use the `PrState` type as well as `PullRequestViewModel.selectedText` and `PullRequestViewModel.selectionList`

Thoughts?

cc: @Br-DiorD

Reply:
@BottleRocket-Colin Bump ;)